src/librustc_ast/ast.rs

```diff
@@ -368,6 +368,7 @@ pub enum GenericParamKind {
         ty: P<Ty>,
         /// Span of the `const` keyword.
         kw_span: Span,
+        default: Option<P<Expr>>,
```

Comment:
Not sure if this is the type we want, so I can change it if there's a better one.